### ISSUE TITLE
Documentation needs some love :heart: 

### ISSUE THREAD

**Issue by laurentS** [SergeyMosin/Appointments#167]
I have tried multiple times to get this appointments app to work, because it looks super promising, but I've still not managed to set it up and book a test meeting with myself :(
I'm obviously dumber than the average person, so I think I need some better explanations than the tutorial does :)

I wouldn't mind opening a PR to improve the tutorial somewhat, but I'm also not sure how to document an app that I struggle to use myself.
Some specific points:
- when I first tried to setup the app, I ended up littering my calendar with dozens of "available" appointments. Eventually I realised that maybe I should use a separate calendar for this (yes, I read the tutorial page, but I guess it didn't make sense to me). Is this what you're supposed to do: have a special calendar that you only use to store the available slots? I think the word "appointment" confused the hell out of me, I think "slots" or something like that would make more sense. To me, an appointment is something that you've agreed on with someone, so it's the output of this app.
- I can't figure out how to make the app take into account that some slots clash with existing appointments: I have my normal calendar in "my cal" with appointments, etc... Now I've added available slots for tuesday and wednesday, say 60min increments from 9am to 5pm to the "free slots" calendar. "my cal" has a booking from 3 to 5, but I can still use appointments to book during that time, resulting in a double booking. I don't understand how to fix that.

**Comment by SergeyMosin** [comment]
RE: I can't figure out how to make the app take into account that some slots clash with existing appointments
Are you using **"Simple"** or **"External"** mode?

**Comment by laurentS** [comment]
> RE: I can't figure out how to make the app take into account that some slots clash with existing appointments
> Are you using **"Simple"** or **"External"** mode?

I think I used "simple" mode.

**Comment by SergeyMosin** [comment]
Currently, only "External" mode can " take into account that some slots clash with existing appointments" in the destination calendar. More info in section 8 of the docs:

> **Source Calendar (External mode)** - Any event with "Show As" a.k.a. "Time As" a.k.a. "Free/Busy" a.k.a. "Time Transparency" set to "Free" (RFC5545 specs "TRANSP:TRANSPARENT") will be available for booking in the public form. Most recurrence rules are supported. Also see Require "Appointment" category.

> **Destination Calendar (External mode)** - Booked appointments will be placed in here. In addition to booked appointments, any events in this calendar marked as "Busy" will prevent conflicting timeslots in the "Source Calendar" from appearing in the public form. Also see Require "Appointment" category.

Just make sure that time slot are as follows:
![alt text](https://user-images.githubusercontent.com/10747559/86515838-f72a5e00-bde9-11ea-936e-0174aa4957e2.jpg)

**Comment by laurentS** [comment]
@SergeyMosin thanks for the tips, I'll give them a try when I get a moment. Would it be useful to put together a simple getting started/tutorial about how to set things up for (what I think is) a typical use case? I feel like the current tutorial is more like a reference.

**Comment by radoeka** [comment]
@laurentS I agree with you.  The apps is nice, the documentation does not show the quality the app has, in my opinion.

BTW I've a dark theme and the documentation does not match it :(

![image](https://user-images.githubusercontent.com/2937666/105628162-8f963780-5e3b-11eb-9f0c-578b01d14d1a.png)

It looks like the explanation for:  "Public Page [online]" -> Pencil 'public page' (3rd option) is missing.  What does it do?

**Comment by Jaxom99** [comment]
Willing to contribute as well. Maybe we could re-work the Readme.md, and restore a FirstSteps/GettingStarted section ? It's linked from the [nextcloud apps public page](https://apps.nextcloud.com/apps/appointments).... 
What would work best for you @SergeyMosin ?

**Comment by Fuseteam** [comment]
i'm very confused how i set my timeslots—
